projects/gfbio-app/src/app/oidc/oidc.component.scss

```diff
@@ -0,0 +1,9 @@
+:host {
+    display: block;
+    padding: 1rem;
+}
+button {
+    width: 20%;
+    margin-left: 40%;
+    margin-right: 40%;
```

Comment:
I guess you need a min-width in `rem` of the button, so that "Logout" fits in there.
Moreover, with `marign: 0 auto;` you can center stuff.

Reply:
I looked over other button usages. Sometimes they are aligned right (e.g. Boxplot), sometimes aligned left (Basic Statistics) and sometimes they are width 100% (Save As in Workspace Settings). 

I don't know if there is a reasoning behind the button placements and widths, but the last one seemed the most similar use case to this, so I styled this in the same manner. d0e929eb0b9cc0d3e987c9dd16658a4479d3f7c8